Formula/sniffglue.rb

```diff
@@ -0,0 +1,20 @@
+class Sniffglue < Formula
+  desc "Secure multithreaded packet sniffer"
+  homepage "https://github.com/kpcyrd/sniffglue"
+  url "https://github.com/kpcyrd/sniffglue/archive/v0.10.1.tar.gz"
+  sha256 "27b9448a317957eface67fab1a264057b03b614b588031e4f47655a8e369ead7"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
+
+    etc.install "sniffglue.conf"
+    man1.install "docs/sniffglue.1"
+  end
+
+  test do
+    system "curl", "-LO", "https://github.com/kpcyrd/sniffglue/raw/master/pcaps/SkypeIRC.pcap"
```

Comment:
Please use a resource for this. You can check with other formula how to.

Reply:
I think I got it to work, can you doublecheck if this is the correct way?